src/pages/faq.astro

```diff
@@ -5,6 +5,22 @@ import Layout from "layouts/Layout.astro";
 import { getSortedFaq } from "scripts/collections";
 
 const faq = await getSortedFaq();
+
+// this sucks, but it's the only way to generate the right data for structured data
+// regardless, this only runs at prerender time so it's okay
+const rawFaqData = await Astro.glob("../content/faq/*.md");
```

Comment:
why is this needed? so you can obtain raw html?

Reply:
correct - Astro collections don't give you access to the raw HTML content (only the component) so we have to use Astro's glob to import them and do it that way